common/OpTestHMC.py

```diff
@@ -314,8 +314,52 @@ def set_lpar_cfg(self, arg_str):
         '''
         if not self.lpar_prof:
             raise OpTestError("Profile needs to be defined to use this method")
-        self.ssh.run_command("chsyscfg -r prof -m %s -p %s -i 'lpar_name=%s,name=%s,%s' --force" %
-                (self.mg_system, self.lpar_name, self.lpar_name, self.lpar_prof,arg_str))
+        self.ssh.run_command("chsyscfg -r prof -m %s -i 'lpar_name=%s,name=%s,%s' --force" %
+                (self.mg_system, self.lpar_name, self.lpar_prof,arg_str))
+
+    def change_to_shared_mode(self, sharing_mode, min_proc_units, desired_proc_units, max_proc_units, overcommit_ratio):
+        self.set_lpar_cfg("proc_mode=shared,sharing_mode=%s,min_proc_units=%s,max_proc_units=%s,"
+                                 "desired_proc_units=%s,min_procs=%s,desired_procs=%s,max_procs=%s" %
+                                (sharing_mode, min_proc_units, max_proc_units, desired_proc_units,
+                                 overcommit_ratio*int(min_proc_units), overcommit_ratio*int(desired_proc_units),
+                                 overcommit_ratio*int(max_proc_units)))
+
+    def change_to_dedicated_mode(self, sharing_mode, min_proc_units, desired_proc_units, max_proc_units):
+        self.set_lpar_cfg("proc_mode=ded,sharing_mode=%s,min_procs=%s,max_procs=%s,desired_procs=%s" %
+                         (sharing_mode, min_proc_units, max_proc_units, desired_proc_units))
```

Comment:
as both commands are same, can these be single method  like change_mode  it can take shared/ded  mode as input